server/hedley/modules/custom/hedley_contact/tests/HedleyContactsByLanguage.test

```diff
@@ -44,6 +44,10 @@ class HedleyContactsByLanguage extends HedleyMunicipalityWebTestCase {
     ];
     $municipality = $this->drupalCreateNode($municipality_settings);
 
+    // Create a User type.
+    $user_types_vocabulary = taxonomy_vocabulary_machine_name_load('user_types');
+    $user_type = $this->createTerm($user_types_vocabulary, 'residents', $he_language_code);
```

Comment:
We have a few terms created on this test, should I change the variable to `$user_type_term` ? (for all of them)